RecoTracker/MkFitCMS/standalone/Geoms/CMS-2017.cc

```diff
@@ -0,0 +1,806 @@
+//-------------------
+// CMS 2017 geometry
+//-------------------
```

Comment:
This will be addressed in a follow-up PR where we will also address generation of mkfit geometry directly from CMSSW. I have already started working on this:
https://github.com/osschar/cmssw/commits/geom-from-fworks

Reply:
I have removed code/data under standalone/CMS-2017/, including the bbs.root binary.

I will restructure standalone/cmssw code to use the same files you mention above. Standalone does not use JSONs.

Reply:
Moved standalone auto-generated files of out this PR (to trackreco/mkFit-external).

Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit: https://github.com/trackreco/cmssw/pull/76/commits/50875516bd74e60edeaba86da070bf8718fcb51c